### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by mwz-cg**

### CODE REVIEW

**Review by Sy1v4in — APPROVED** [review]

**Review by BeLi4L — APPROVED** [review]

File: markdowns/welcome.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The source code is on [GitHub](https://github.com/CodinGame/rust-template), plea
 
 # Hands-on Demo
 
-@[Luke, how many stars are there in these galaxies?]({ "stubs": [ "src/lib.rs" ], "command": "cargo test -q --test test test_universe -- --nocapture" })
+@[Luke, how many stars are there in these galaxies?]({ "stubs": [ "src/lib.rs" ], "buildCommand": "cargo build", "command": "cargo test -q --test test test_universe -- --nocapture" })
```
Comment (Sy1v4in) [inline markdowns/welcome.md]: There is no `buildCommand` in the markdown and you already have one in the `techio.yml` file.

Comment (mwz-cg) [reply markdowns/welcome.md]: Wait there is no buildCommand? I am so confused right now :confused:

Comment (mwz-cg) [reply markdowns/welcome.md]: Ok I got it, fixed in 7bf9c9c89cc551427f930ee6ff7a4c5c7286f615

**Comment by mwz-cg** [comment]
Rebased on master, ready to merge

**Pull request merged**